### PR TITLE
Include Radio Jambo data for e03

### PR DESCRIPTION
**Pull request by as2388**
Just to explain what’s going on: We ran 4 weeks of shows on Radio Jambo (COVID19) with limited control over the content, so asked the same question each week. That project has now come to an end. Separately, we were running a project targeting urban poor (COVID19-KE-Urban) on Ghetto Radio, which is continuing. However, we now have funding to control the content on Jambo, so we’re harmonising the content between the two stations. This can be thought of as effectively taking KE-Urban nationwide, so we only need one dataset in Coda (same question, same coding frame) and one analysis file, which in future may be tagged with the shortcode each message was received on. We’re still maintaining the two shortcodes, because of the risk to engagement of trying to change the codes now, and because of the risk to analysis of not being able to separate the radio stations in future should we need to, but are now pulling the data from both instances into this pipeline. That’s why you see the configuration file updated to refer to both instances. 

For the most part this “just works”, even if people respond to the RQA on both shortcodes. The main exception is demogs, where if people answer both we’ll only look at the answers from one. The overlap is currently very low (5 common numbers after 4000 vs. 300 participants on jambo/ghetto so far), and we have alternative strategies like concatenate or TextIt contact fields sync available if needed.

One final piece of complexity: we switched to the covid19_s01e02 activation flow before taking the decision to harmonise the projects. We then renamed this to s01e03, but there are still some runs with the old terminology, which is why you see the remapping from “Rqa_S01E02 (Text) - covid19_s01e03_activation”.

TL;DR: COVID19-KE-Urban is now running on Ghetto and Jambo but on different shortcodes. This PR pulls the newest Jambo data into the KE-Urban pipeline too.

### CODE REVIEW

**Review by IsaackMwenda — APPROVED** [review]
This LGTM! -  also check if urban ProjectStartDate covers demogs or rqa data from covid19

**Comment by as2388** [comment]
> This LGTM! - also check if urban ProjectStartDate covers demogs or rqa data from covid19

Yep, good to think about. All the start/end dates do is prevent us from needing to label irrelevant messages if we have an activation flow set-up before we ask the first question or for a long time after the last one. For COVID19-2: the start date given here matches when we went live with a promo advertising that shortcode, so is correct. For COVID19 - the instance was running earlier, but the activation flow for e03 was not. All of the data in that flow is relevant so we're fine as long as the start date is older than that flow, which it is.
For demogs - the pipeline only actually picks up demogs for people who sent an activation message within the time window, so no need to worry about start/end dates here. It's advantageous for us to be able to pick up demogs from outside the activation flow window.
